README.md

```diff
@@ -66,14 +66,15 @@ In addition, it's important that all members of the team (including any and all
 <a name="play-2"></a>
 ### 2 // Identify Problems
 
-Working with the state and external partners, determine where the largest sources of failures are occurring while people are renewing. Where are people being moved from automatic renewal to manual renewal? Where are people struggling or failing when completing a renewal? What methods are most commonly utilized, and what are the biggest problems with those methods?
+Working with the state and external partners, determine the largest sources of administrative burden and churn during renewal. Where are people being moved from automatic renewal to manual renewal? Where are people struggling or falling off when completing a renewal? What methods are most commonly utilized, and what are the biggest problems with those methods?
 
 When determining the problems, it's important to keep the focus on the “end-to-end” renewal process. An ex parte attempt is only the first step in a person's renewal. For the millions of people manually renewing, there are vast opportunities to improve the experience and increase renewal rates.
 
 * **Pull the numbers.** Pull end-to-end funnel data, including where drop-offs occur at each step in the renewal process. Pull ex parte failure reasons, and organize by the number of people impacted. Pull web traffic numbers to understand how people are accessing online renewal materials.
 * **Be a renewer.** Put your renewer hat on and do some Googling to figure out how to renew Medicaid in your state. Do this on mobile, record screenshots as you go, and note pain points throughout the process. Call customer support when you get stuck, take notes on the IVR and overall experience.
 * **Learn from experts.** Meet with Navigators and eligibility workers to identify top challenges across the renewal process that members of their community experience. Meet in smaller groups with state SMEs and decision makers to better understand the challenges they've been dealing with.
 * **Map the experience.** Map a high-level user journey of the renewal end-to-end experience, noting common pain points and workarounds.
+* **Don't split the party** - don't separate into ex parte and manual teams. It's a safe bet the backend vendor engineer has never watched an eligibility worker do thier job: your team will need to have that context to help them identify root causes. Common issues reported by Navigators may point to problems that could be solved in the ex parte phase. Approach the system from a holistic, human-centered perspective. 
```

Comment:
I might rephrase this second sentence as something like:

> It's possible that the implementation teams have not had the opportunity to witness an eligibility worker process a renewal; your team...

It removes any potential implications about vendors!

Reply:
done